packages/manager/apps/telecom/src/app/common/translations/Messages_en_GB.json

```diff
@@ -197,7 +197,7 @@
   "common_home": "Home",
   "unit_bits_per_sec": "{{val}} bps",
   "unit_kilo_bits_per_sec": "{{val}} kB/s",
-  "unit_mega_bits_per_sec": "{{val}} MB/s",
+  "unit_mega_bits_per_sec": "{{val}} Mb/s",
```

Comment:
Should this be applied on all units of size?

Reply:
I have raised a new TRANS request for the modification of other units as well.